packages/teams-js/src/public/meeting.ts

```diff
@@ -925,6 +926,47 @@ export namespace meeting {
     );
   }
 
+  /**
+   * This function is used to join a meeting
+   * @param joinMeetingParams
```

Comment:
Please use `@link` in this comment to connect this param to the `JoinMeetingParams` interface. That will create a hyperlink from this parameter to its type in the dev docs.

Reply:
Added the @link to JoinMeetingParams